src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -54,6 +54,7 @@
    and request handling during normal Puppet operation.
    Most of these are Puppet configuration settings."
   {:access-control           (schema/maybe AccessControl)
+   :allow-authorization-extensions  schema/Bool
```

Comment:
We maintain this list here and in `ca_testutils.clj`, in ca_testutils we updated the column spacing. We might as well do that here. Or not align the spacing at all, fine either way.

Reply:
I can align them (I saw the manage-file-internal-positions below was not aligned, so I didnt bother here).  BUt I'll fix 'em up.